wisdem/drivetrainse/drive_structure.py

```diff
@@ -207,8 +210,12 @@ def compute(self, inputs, outputs, discrete_inputs, discrete_outputs):
         m_hub = float(inputs["hub_system_mass"][0])
         cm_hub = float(inputs["hub_system_cm"][0])
         I_hub = inputs["hub_system_I"]
-        F_hub = inputs["F_hub"]
-        M_hub = inputs["M_hub"]
+        F_aero_hub = inputs["F_aero_hub"]
+        M_aero_hub = inputs["M_aero_hub"]
+        blades_mass = inputs['blades_mass']
+        F_hub = F_aero_hub
+        F_hub[2] -= blades_mass * gravity
```

Comment:
Instead of adding the load this way, it is better practice with Frame3DD to add a point mass.  I can go in and make that small change.